MatFilesSINR/computeChannelGain.m

```diff
@@ -27,8 +27,8 @@
     if simParams.typeOfScenario==constants.SCENARIO_URBAN %ETSI-Urban
         D1 = zeros(Nvehicles,Nvehicles);
         C = zeros(Nvehicles,Nvehicles);
-        horizontal=abs(real(positionManagement.direction));
-        vertical=abs(imag(positionManagement.direction));
+        horizontal=cos(positionManagement.direction);
```

Comment:
the original lines here will generate a non negative num. I think the following lines would use this kind of number.